src/store.js

```diff
@@ -0,0 +1,23 @@
+
+import {
+  createStore, applyMiddleware, combineReducers, compose,
```

Comment:
one item per line is more readable structure

Reply:
Modified